src/app/containers/StoryPromo/LinkContents.jsx

```diff
@@ -15,10 +15,10 @@ const LinkContents = ({ item }) => {
   } = useContext(ServiceContext);
 
   const isMedia = deepGet(['cpsType'], item) === 'MAP';
-  const headline = deepGet(['headlines', 'headline'], item);
-
+  const headlines = deepGet(['headlines'], item);
+  const { headline, overtyped } = headlines;
   if (!isMedia) {
-    return headline;
+    return overtyped || headline;
```

Comment:
Could we extract this to a variable like `contents` and use it on line `38` and `21`